CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to Raygun4ReactNative
+
+## Project and library organisation
+
+Building the project requires [Node.js](https://nodejs.org).
```

Comment:
it's been a while for me but is this correct?

Reply:
Yes, you need npm and other tools that are part of nodejs to setup the development tools